feature-group-definitions/array-fill.dist.yml

```diff
@@ -0,0 +1,27 @@
+# Generated from: array-fill.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Array fill()
+description: The `fill()` method of arrays and typed arrays sets all or some items of an array to a given a value.
+spec:
+  - https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.fill
+  - https://tc39.es/ecma262/multipage/indexed-collections.html#sec-%25typedarray%25.prototype.fill
+snapshot: ecmascript-2015
+group:
+  - arrays
+  - typed-arrays
+status:
+  baseline: high
+  baseline_low_date: 2016-03-21
```

Comment:
This changed due to the Firefox and Safari versions changing, because `fill()` didn't ship on array and typed arrays at the same time. @Elchi3 how do you think we should handle this?